src/Store/selector.js

```diff
@@ -0,0 +1,3 @@
+export function checkedAndUnchecked(state) {
+  return !state.isChecked && state.wasChecked;
+}
```

Comment:
`selector.js` gets derived state.

If you just need simple state, you can get it straight from the `state` global: `state.isChecked`